mmdet/models/layers/msdeformattn_pixel_decoder.py

```diff
@@ -99,9 +77,9 @@ def __init__(self,
             input_conv_list.append(input_conv)
         self.input_convs = ModuleList(input_conv_list)
 
-        self.encoder = build_transformer_layer_sequence(encoder)
-        self.postional_encoding = build_positional_encoding(
-            positional_encoding)
+        self.encoder = Mask2FormerTransformerEncoder(**encoder)  # TODO: Notice reference_point
```

Comment:
remember this todo

Reply:
@jshilong The encoder has been customized for Mask2Former. This will be rethinked when refactoring Mask2Former.